bindings/python/examples/dnn/CMakeLists.txt

```diff
@@ -4,11 +4,11 @@ project(aditof-python-dnn)
 
 # Download MobileNetSSD_deploy prototxt and caffemodel
 set(PROTOTEXT_FILE "MobileNetSSD_deploy.prototxt")
-set(PROTOTEXT_URL "https://raw.githubusercontent.com/chuanqi305/MobileNet-SSD/f5d072ccc7e3dcddaa830e9805da4bf1000b2836/MobileNetSSD_deploy.prototxt")
```

Comment:
I got MobileNetSSD_deploy.prototxt and MobileNetSSD_deploy.caffemodel from a different github repository, because the files that are obtained from the existing repository are not working.